### PR TITLE
decompile rom_1068C.s and rom_528AC.s

### PR DESCRIPTION
**Pull request by camthesaxman**

### CODE REVIEW

File: src/rom_3219C.c [hunk 1]
```diff
@@ -43,31 +43,31 @@ void sub_3219C(void)
         {
             s16 numSpecialMons = 0;
             gUnknown_020314E0->catchModeSpecies = 0;
-            if (gUnknown_0200B134[SPECIES_AERODACTYL])
+            if (gMain+0x74[SPECIES_AERODACTYL])
```
Comment (huderlem) [inline src/rom_3219C.c]: Did you forget to clean this up?  This should be `gMain.saveData.pokedexFlags[SPECIES_AERODACTYL]`

Comment (camthesaxman) [reply src/rom_3219C.c]: It doesn't match with gMain. I'm beginning to think that saveData is not part of gMain, but then again, there's code that accesses gMain.unk2E8 and gMain.spriteGroups, so I think it must be.

Comment (huderlem) [reply src/rom_3219C.c]: Do you want to play around more with that, or should I merge as-is?

Comment (camthesaxman) [reply src/rom_3219C.c]: Merge it as-is. I'll play around with it once we have a better understanding of gMain.

**Pull request merged**